mreg_cli/network.py

```diff
@@ -236,38 +236,45 @@ def find(args: Namespace):
     """List networks matching search criteria."""
     args_dict = vars(args)
 
-    params = {}
-    param_names = [
-        "network",
-        "description",
-        "vlan",
-        "dns_delegated",
-        "category",
-        "location",
-        "frozen",
-        "reserved",
-    ]
-    for name in param_names:
-        value = args_dict.get(name)
-        if value is None:
-            continue
-        param, val = convert_wildcard_to_regex(name, value)
-        params[param] = val
-
-    if not params:
-        cli_warning("Need at least one search criteria")
-
-    path = f"/api/v1/networks/"
-    networks = get_list(path, params)
+    ip_arg = args_dict.get("ip")
+    if not ip_arg:
+        params = {}
+        param_names = [
+            "network",
+            "description",
+            "vlan",
+            "dns_delegated",
+            "category",
+            "location",
+            "frozen",
+            "reserved",
+        ]
+        for name in param_names:
+            value = args_dict.get(name)
+            if value is None:
+                continue
+            param, val = convert_wildcard_to_regex(name, value)
+            params[param] = val
+
+        if not params:
+            cli_warning("Need at least one search criteria")
+
+        path = f"/api/v1/networks/"
+        networks = get_list(path, params)
+    else:
+        ip_range = get_network_range_from_input(ip_arg)
+        network_info = get_network(ip_range)
+        networks = [network_info]
```

Comment:
Classic double negative approach. I'd swap the order and skip the `not`.

Reply:
[cf93a3b](https://github.com/unioslo/mreg-cli/pull/159/commits/cf93a3b1faba8dcd4089516f3dc174e56e7dd352)